buildkite/src/Jobs/Release/ArchiveNodeArtifact.dhall

```diff
@@ -27,6 +27,8 @@ Pipeline.build
     spec =
       JobSpec::{
         dirtyWhen = [
+          S.strictlyStart (S.contains "buildkite/src/Jobs/Release/ArchiveNodeArtifact"),
+          S.exactly "buildkite/nightly" "txt",
```

Comment:
Yea seems clearer to trigger on a Job versus some txt file tho up in the air for me.